specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/stable/2022-05-01/ResourceHealth.json

```diff
@@ -341,6 +341,174 @@
+          {
+            "$ref": "#/parameters/ImpactedResourceNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The details of Impacted Resource.",
+            "schema": {
+              "$ref": "#/definitions/eventImpactedResource"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ImpactedResourcesGet": {
+            "$ref": "./examples/ImpactedResources_GetByTenantId.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ResourceHealth/events/{eventTrackingId}/listSecurityAdvisoryImpactedResources": {
+      "post": {
+        "tags": [
+          "SecurityAdvisoryImpactedResources"
+        ],
+        "operationId": "SecurityAdvisoryImpactedResources_ListBySubscriptionIdAndEventId",
+        "description": "Lists impacted resources in the subscription by an event (Security Advisory).",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/EventTrackingIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/FilterParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The list of impacted resources in the subscription by an event (Security Advisory).",
+            "schema": {
+              "$ref": "#/definitions/eventImpactedResourceListResult"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "ListSecurityAdvisoryImpactedResourcesBySubscriptionId": {
+            "$ref": "./examples/SecurityAdvisoryImpactedResources_ListBySubscriptionId_ListByEventId.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.ResourceHealth/events/{eventTrackingId}/listSecurityAdvisoryImpactedResources": {
+      "post": {
+        "tags": [
+          "SecurityAdvisoryImpactedResources"
+        ],
+        "operationId": "SecurityAdvisoryImpactedResources_ListByTenantIdAndEventId",
+        "description": "Lists impacted resources in the tenant by an event (Security Advisory).",
+        "parameters": [
+          {
```

Comment:
Post action needs to have a request body. No filter query parameters are allowed.

This is applicable in all places in this file